gammapy/maps/geom.py

```diff
@@ -495,6 +497,8 @@ def from_nodes(cls, nodes, **kwargs):
             raise ValueError("Nodes array must have at least one element.")
         if len(nodes) != len(np.unique(nodes)):
             raise ValueError("MapAxis: node values must be unique")
+        if ~(np.all(nodes == np.sort(nodes)) or np.all(nodes[::-1] == np.sort(nodes))):
```

Comment:
Maybe move this statement to `MapAxis.__init__()` to avoid the duplication in `from_edges()` and `.from_nodes()`?

Reply:
Thanks @adonath ! Done